examples/seismic/model.py

```diff
@@ -722,4 +797,99 @@ def critical_dt(self):
         #
         # The CFL condtion is then given by
         # dt < h / (sqrt(2) * max(vp)))
-        return self.dtype(.5*mmin(self.spacing) / (np.sqrt(2)*mmax(self.vp)))
+        # FIXME: Fix 'Constant' so that that mmax(self.vp) returns the data value
+        return self.dtype(.5*mmin(self.spacing) / (np.sqrt(2)*mmax(self.vp.data)))
+
+
+class ModelViscoelastic(GenericModel):
```

Comment:
Wouldn't it be easier to inherit from Elastic model and only setup the extras parameters?

Reply:
Yeah, was thinking about doing it that way earlier. Will try and remember if there's a reason I didn't immediately do that.